doc/conf.py

```diff
@@ -29,13 +29,15 @@
     "sphinx.ext.napoleon",
     "sphinx_gallery.gen_gallery",
     "sphinx_copybutton",
+    "sphinx_design",
 ]
 
 # Autosummary pages will be generated by sphinx-autogen instead of sphinx-build
 autosummary_generate = []
 
 # Auto-generate header anchors with MyST parser
 myst_heading_anchors = 4
+myst_enable_extensions = ["colon_fence"]
```

Comment:
Document a bit what this does (enable `::::` code fences). https://myst-parser.readthedocs.io/en/v0.17.2/syntax/optional.html#code-fences-using-colons

```suggestion
# Allow code fences using colons
myst_enable_extensions = ["colon_fence"]
```